packages/preferences/src/browser/util/preference-tree-generator.ts

```diff
@@ -136,8 +136,18 @@ export class PreferenceTreeGenerator {
         return split;
     }
 
-    private formatString(string: string): string {
-        const specifier = this.split(string);
-        return specifier.map(word => word.slice(0, 1).toLocaleUpperCase() + word.slice(1)).join(' ').trim();
+    protected formatString(string: string): string {
+        let formatedString = string[0].toLocaleUpperCase();
+        for (let i = 1; i < string.length; i++) {
+            if (this.isUpperCase(string[i]) && string[i - 1] !== ' ' && !this.isUpperCase(string[i - 1])) {
```

Comment:
Minor: I think this is fine as is - I'll approve it momentarily - but the check `!== ' '` could be generalized to regex `/\s/`, since not all scripts use the ' ' character.